code/game/turfs/simulated/walls.dm

```diff
@@ -105,14 +110,16 @@
 
 /turf/simulated/wall/attack_paw(mob/user as mob)

```

Comment:
There's no reason for the hulk code to be under this at all, actually.
Hulk is already checked in attack_hand.